docs_src/microservices/configuration/CommonConfiguration.md

```diff
@@ -98,13 +98,22 @@ The tables in each of the tabs below document configuration properties that are
     |Property|Default Value|Description|
     |---|---|---|
     |||Each service has it own collect of Clients that it uses|
-    |Protocol | http  | The protocol to use when building a URI to local the service endpoint|
-    |Host | localhost  | The host name or IP address where the service is hosted |
+    |Protocol | http | The protocol to use when building a URI to local the service endpoint|
+    |Host | localhost | The host name or IP address where the service is hosted|
     |Port | 598xx | The port exposed by the target service|
+    |UseMessageBus | false | indicate whether to use Messaging version of client |
+    |Topics |  | holds the MessageBus Topics used by the client to communicate to the service|
+    || CommandRequestTopicPrefix = edgex/core/command/request | for publishing the internal command request|
```

Comment:
If only core-command uses this and QueryResponseTopic (?) in its configuration, should probably say so.

Reply:
The Clients configuration here means other EdgeX service (except core-command) will try to connect to core-command and use its APIs.

Reply:
Yes, this seems at first to be Core Command specific, but any service that is using Command Client needs this common configuration.

Reply:
OK, I see better now that difference between "common" and "Ch-Command" parameters.